.github/workflows/test.yaml

```diff
@@ -214,3 +216,47 @@ jobs:
         run: |
           echo "::error::unexpected status code: got: ${{ steps.lava.outputs.status }}, want: ${{ env.WANT_STATUS }}"
           exit 1
+
+  test-no-vulns:
+    env:
+      WANT_STATUS: 0 # ExitCodeInfo
```

Comment:
[ExitCodeInfo](https://github.com/adevinta/lava/blob/a7eabad5fcf9c781bc2dc296606eea121917b69f/internal/report/report.go#L309) is `100`.

```suggestion
      WANT_STATUS: 0 # No vulnerabilities found
```

Reply:
done